src/views/ModelPlan/TaskList/ITSolutions/_components/CheckboxCard/index.tsx

```diff
@@ -84,26 +147,55 @@ const CheckboxCard = ({
         />
       )}
 
-      <Card className={classNames(className)}>
-        <div className="solutions-checkbox__header padding-2">
-          <Field
-            as={Checkbox}
-            disabled={disabled}
-            id={id}
-            testid={id}
-            name={`solutions[${index}].needed`}
-            label={t('selectSolution')}
-            value={!!solution.needed}
-            checked={!!solution.needed}
-          />
-
-          <h3 className="margin-y-2">{solution.nameOther || solution.name}</h3>
-
-          <div className="margin-bottom-2 solutions-checkbox__body-text">
-            {solutionMap &&
-              hk(`solutions.${solutionMap.key}.about.description`)}
+      <Card
+        className={classNames('width-full', className)}
+        containerProps={{ className: 'padding-3 flex-justify' }}
+      >
+        <div className="solutions-checkbox__above-the-border">
+          <div className="solutions-checkbox__header">
+            <Field
+              as={Checkbox}
+              disabled={disabled}
+              id={`${id}-${solution.id}`}
+              name={`solutions[${index}].needed`}
+              label={t('selectSolution')}
+              value={!!solution.needed}
+              checked={!!solution.needed}
+              className="margin-y-0"
+            />
+
+            {/* If solution is one of the treatAsOther, then render the following  */}
+            {solution.key &&
+              treatAsOtherSolutions.includes(solution.key) &&
+              !isDefaultSolutionOptions && (
+                <>
+                  <h3 className="margin-top-2 margin-bottom-0">
+                    {solution.otherHeader}
+                  </h3>
+                  <h5 className="text-normal margin-top-0 margin-bottom-2">
+                    {translateOperationalSolutionKey(solution.key)}
+                  </h5>
+                </>
+              )}
```

Comment:
This conditional doesn't all for a rendering of both a default solution and a custom solution, which is what `Contractor` is for `Helpdesk support`